src/ESLint.Formatter/test/sarif.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * @fileoverview Tests for SARIF format.
```

Comment:
>Tests [](start = 17, length = 5)

where are these tests running? how do we run these tests? can we automate their execution? or minimally document execution process